views/package_suppliers.dt

```diff
@@ -0,0 +1,37 @@
+extends layout
+
+block title
+	- auto title = "Package suppliers";
+
+block body
+	h2 Introduction
+
+	p Dub supports package suppliers of type Dub Registry, Maven and File System. By default Dub will search on Dub Registry <a href="https://code.dlang.org">https://code.dlang.org</a> and various fallbacks.
+
+	p Package suppliers can be specified either using argument <code>--registry</code> or using setting <code>registryUrls</code>. The type of the package supplier could be specified using a prefix <code>dub+</code>, <code>mvn+</code> or <code>file://</code>. By default type Dub Registry is assumed.
```

Comment:
link "setting" to the settings.html page

Reply:
```suggestion
	p Package suppliers can be specified either using argument <code>--registry</code> or using setting <code>registryUrls</code>. The type of the package supplier can be specified by prefixing the URL with <code>dub+</code> for type Dub Registry, <code>mvn+</code> for type Maven or using the <code>file://</code> scheme for type File System. Without a prefix, the type Dub Registry (<code>dub+</code>) is assumed.
```

Reply:
fixed